src/ocm/components/clusterConfiguration/DiscoveryImageSummary.tsx

```diff
@@ -10,13 +10,10 @@ type DiscoveryImageSummaryProps = {
 };
 
 const DiscoveryImageSummary: React.FC<DiscoveryImageSummaryProps> = ({ cluster, ...restProps }) => {
-  const { imageUrl, error, isLoading } = useInfraEnvImageUrl(cluster.id);
+  const { imageUrl, error } = useInfraEnvImageUrl(cluster.id);
   if (error) {
     return <ErrorState />;
   }
-  if (isLoading) {
-    return <LoadingState />;
-  }
```

Comment:
why did you remove loading state ?

Reply:
This change is from https://github.com/openshift-assisted/assisted-ui-lib/pull/1327